src/modules/iotjs_module_adc.h

```diff
@@ -34,7 +34,8 @@ typedef struct {
 
 #if defined(__linux__)
   iotjs_string_t device;
-#elif defined(__NUTTX__)
+#endif
+#if defined(__NUTTX__) || defined(__TIZENRT__)
```

Comment:
It is not any better. You are suggesting introducing dependency between not related lines. Why we should change TizenRT/Nuttix code if someone will decide that linux structure is not necessary or remove support for Linux?  An #elseif is not faster because precompiler generates header during compilation not at runtime, there is not final else with #error so there is not point in creating any relation between precompiler directives for Linux and Tizen.